### ISSUE TITLE
OptimizationFunction Definition

### ISSUE THREAD

**Issue by ugccelik** [ChrisRackauckas/universal_differential_equations#49]
Hello,

I was investigating the LotkaVolterra problem scripts and i saw that the optimization function is defined as follows;

optf = Optimization.OptimizationFunction((x,p)->loss(x), adtype)

i am wondering why loss function is given to optimization-function definition as : (x,p)->loss(x) in here while it is only depending on ''p'' ? Since, it isn't working when it is written as loss(p).

**Comment by ChrisRackauckas** [comment]
That `p` is the hyperparameters of the optimization, not the parameters of the model.

**Comment by ugccelik** [comment]
Thank you for your response. Is there any documentation about these hyper-parameters of the optimization, where can i find them. Since documentation about OptimizationFunction don't mention anything about it.

**Comment by ChrisRackauckas** [comment]
Good point. https://github.com/SciML/SciMLBase.jl/pull/243
